### ISSUE TITLE
How to use non-local modules

### ISSUE THREAD

**Issue by yehuixie** [CMU-CREATE-Lab/deep-smoke-machine#13]
How to use non-local modules.
It seems that simply using `python train.py i3d-nl-rgb-cv-1` does not work.

**Comment by yenchiah** [comment]
What kind of error messages did you get?

**Comment by yehuixie** [comment]
> What kind of error messages did you get?

I got no error messages.
But I found that the method `def add_nl_to_i3d(self)` did not work in `pytorch_i3d_nl.py`.
I checked the `i3d-grb-cv-1` model and the `i3d-nl-rgb-cv-1` model and it turns out that both models have the same number of parameters.
So I wonder if I am not using the non-local module correctly.

**Comment by yenchiah** [comment]
Did you see the printed message from [this line of code](https://github.com/CMU-CREATE-Lab/deep-smoke-machine/blob/0ded01e4683f0ed6068841d05e9545a65778edc6/back-end/www/model/pytorch_i3d_nl.py#L60)?

**Comment by yenchiah** [comment]
Thank you so much for bringing the issue with the code to my attention. After examining it, I found that there was indeed an error in using the non-local module.

To correct this error, I have pushed a commit. I will also re-run the model to correct the F1-scores in the RGB-NL model in the paper (in Table 7). The corrected version of the paper will be uploaded to arXiv to override our previously submitted preprint script.

Thank you again for your valuable feedback.
